include/fastrtps/attributes/RequesterAttributes.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file RequesterAttributes.h
+ */
+
+#ifndef REQUESTERATTRIBUTES_H_
+#define REQUESTERATTRIBUTES_H_
+
+#include "PublisherAttributes.h"
+#include "SubscriberAttributes.h"
+
+namespace eprosima {
+namespace fastrtps {
+
+class RequesterAttributes
+{
+public:
+    RequesterAttributes() = default;
+
+    bool operator==(const RequesterAttributes& b) const
+    {
+        return (this->service_name == b.service_name) &&
```

Comment:
Does not `reply_type` and `request_type` check?

Reply:
It is not necessary because they are set at `publisher.topic.topicDataType` and `subscriber.topic.topicDataType`, so it is checked from `publisher` and `subscriber`.